frontend/components/core/ReTableInfo.vue

```diff
@@ -226,6 +236,22 @@ export default {
       type: String,
       default: undefined,
     },
+    emptyTitle: {
+      type: String,
+      default: undefined,
+    },
+    emptyFilteredTitle: {
+      type: String,
+      default: undefined,
+    },
+    emptyDescription: {
+      type: String,
+      default: undefined,
+    },
+    emptyIcon: {
+      type: String,
+      default: undefined,
+    },
```

Comment:
We can separate props in 2 blocks:

- Info when no data
- Info when component search has no results

````javascript
noDataInfo: {
   title: "",
   description: "", // maybe message is a better naming?
   icon: "",
},
emptySeachInfo: {
   title: "",
   description: "", // maybe message is a better naming?
   icon: "",
},
````